.circleci/config.yml

```diff
@@ -23,15 +23,16 @@ jobs:
   build:
     working_directory: ~/build
     docker:
-      - image: rust:1.21.0
+      - image: rust:latest
```

Comment:
You can switch the image to `rustlang/rust:nightly` instead to avoid having to set up a toolchain down below.

Reply:
Thanks! I'm rather unfamiliar with circleci's images.